src/hotspot/share/gc/shared/space.hpp

```diff
@@ -312,11 +312,6 @@ class CompactPoint : public StackObj {
 // The compaction operations are implemented by the
 // scan_and_{adjust_pointers,compact,forward} function templates.
 // The following are, non-virtual, auxiliary functions used by these function templates:
-// - scan_limit()
-// - scanned_block_is_obj()
-// - scanned_block_size()
-// - adjust_obj_size()
-// - obj_size()
```

Comment:
The entire surrounding comment needs to be updated now that these functions have been removed.